### PR TITLE
update certifcate for v2.0.6

### PR DESCRIPTION
**Pull request by ganchdev**
Upgrades Acme Manager to satisfy the new version of Acme Client.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'json'
 gem 'acme-client'
+gem 'json'
```
Comment (catphish) [inline Gemfile]: This change probably isn't necessary.

File: lib/acme_manager/certificate.rb [hunk 2]
```diff
@@ -37,43 +37,44 @@ def renew
...
+      csr.sign(private_key, OpenSSL::Digest::SHA256.new)
+      order.finalize(:csr => csr)
+
       FileUtils.mkdir_p(File.join(AcmeManager.data_path, 'certificates', domain))
-      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'key.pem'), certificate.request.private_key.to_pem)
-      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'cert.pem'), certificate.to_pem)
-      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'chain.pem'), certificate.chain_to_pem)
+      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'key.pem'), private_key.to_pem)
+      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'cert.pem'), order.certificate)
+      File.write(File.join(AcmeManager.data_path, 'certificates', domain, 'chain.pem'), order.certificate)
```
Comment (catphish) [inline lib/acme_manager/certificate.rb]: I don't see any reason to write this file, just abandon it, nothing should use it.

**Review by catphish — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (ganchdev) [reply Gemfile]: That was probably rubocop rearranging names..

File: lib/acme_manager/certificate.rb (hunk 2 above)
Comment (ganchdev) [reply lib/acme_manager/certificate.rb]: Okay, will do.

**Pull request merged**